CHANGELOG.md

```diff
@@ -7,6 +7,20 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [2.9.1] - 2023-07-06
```

Comment:
We could also use 2.9.0 since there is no release for that and no changes to application code after that, but might bit of a bad practice to move that git tag forward

Reply:
True, didn't want to mess with the tags as the v2.9.0 has been there already for a while.